enos/modules/vault_verify_version/templates/verify-cluster-version.sh

```diff
@@ -21,25 +20,24 @@ test -x "$binpath" || fail "unable to locate vault binary at $binpath"
 export VAULT_ADDR='http://127.0.0.1:8200'
 export VAULT_TOKEN='${vault_token}'
 
-if [[ "$builddate" != "" ]]; then
-  build_date=$builddate
-else
+if [[ "$build_date" == "" ]]; then
```

Comment:
This will give a false correct (pass) if the `build_date` is missed during the build, to validate it against what is built, the `build_date` was passed as an input from `build.yml`